pkg/setting/setting.go

```diff
@@ -560,6 +563,9 @@ func NewConfigContext(args *CommandLineArgs) error {
 	authBasic := Cfg.Section("auth.basic")
 	BasicAuthEnabled = authBasic.Key("enabled").MustBool(true)
 
+	// global plugin settings
```

Comment:
Will fix the capitlisation.

Reply:
On second thoughts, this is using the same capitalisation as its neighbours so I'll leave it as-is.